managed/services/backup/backup_service.go

```diff
@@ -217,10 +218,16 @@ type prepareRestoreJobParams struct {
 	ServiceType   models.ServiceType
 	DBConfig      *models.DBConfig
 	DataModel     models.DataModel
+	PITRTimestamp time.Time
 }
 
 // RestoreBackup starts restore backup job.
-func (s *Service) RestoreBackup(ctx context.Context, serviceID, artifactID string) (string, error) {
+func (s *Service) RestoreBackup(ctx context.Context, serviceID, artifactID string, pitrTimestamp time.Time) (string, error) {
+	err := s.checkArtifactModePreconditions(ctx, artifactID, pitrTimestamp)
+	if err != nil {
+		return "", err
+	}
```

Comment:
Short form works well for such cases
```suggestion
	if err := s.checkArtifactModePreconditions(ctx, artifactID, pitrTimestamp); err != nil {
		return "", err
	}
```